maquetado/scss/styles.scss

```diff
@@ -81,3 +118,135 @@
 .height {
   line-height: 24px;
 }
+
+.input-description {
+  font-size: 16px;
+}
+
+.input-field {
+  background-color: $white;
+  border-radius: 5px;
+  height: 30px;
+  margin-bottom: 30px;
+  width: 250px;
+}
+
+.login-container {
+  display: flex;
+  justify-content: center;
+}
+
+.info-container {
+  background-color: $wild-sand;
+  justify-content: space-between;
+  left: 50%;
+  max-width: 300px;
+  padding: 25px 25px 15px;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 1000%;
+  &::before {
+    content: '';
+    background-color: $cerulean;
+    height: 8px;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+}
+
+.navbar {
+  background-color: $white;
+  box-shadow: 0 4px 10px 0 $black-05;
+  height: 70px;
+  padding-left: 200px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  &::before {
+    content:'';
+    background-color: $cerulean;
+    height: 4px;
+    left: 0;
+    position: absolute;
+    width: 100%;
+  }
+}
+
+.navbar-logo {
+  height: 70px;
+  width: 210px;
+}
+
+.redirect-arrow {
+  font-size: 20px;
+  margin-left: 100px;
+  position: relative;
+  &::before {
+    content: '';
+    background: url(./assets/back-arrow.png) no-repeat;
+    height: 40px;
+    margin-left: -40px;
+    position: absolute;
+    width: 100%;
+    left: 0px;
+  }
+}
+
+.login-img {
+  height: 70px;
+  margin: 0 24px 30px;
```

Comment:
This shouldn't be necessary. You can use `margin: 0 auto;` to center this image or you could use flex for this, justify-content: center in this case since you are already using column for the form.